test/integration/admin/api/buyers_users_controller_test.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class Admin::Api::BuyersUsersControllerTest < ActionDispatch::IntegrationTest
```

Comment:
I do not think it is related to this controller

Please create a more generic one to test it ;)
Btw no need to test in controller just make a rack test is enough

Reply:
Awesome, will do 👍